src/visions/application/summaries/series/unique_summary.py

```diff
@@ -21,13 +20,15 @@ def unique_summary_complex(series: pd.Series) -> dict:
     Args:
         series: series to summarize
 
+    Notes:
+        Until complex bug is fixed:
+        - https://github.com/pandas-dev/pandas/issues/17927
+        - https://github.com/pandas-dev/pandas/pull/27599
+
     Returns:
 
     """
-    summary = {}
-    # Until complex bug is fixed:
-    # https://github.com/pandas-dev/pandas/issues/17927
-    # https://github.com/pandas-dev/pandas/pull/27599
+
     n_unique = len(set(series[series.notna()].values))
-    summary.update({"n_unique": n_unique})
+    summary = {"n_unique": n_unique}
```

Comment:
any reason not to use `nunique()`?

i.e. 

```
summary = {"n_unique": series.nunique(dropna=True)}
```

Reply:
This function is specific to complex types, where there is a bug in `nunique()`.